routesrv/polling.go

```diff
@@ -90,6 +72,9 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 		case routesCount == 0:
 			log.Error(LogRoutesEmpty)
 
+			p.metrics.UpdateGauge("routes_count", 0)
```

Comment:
Lets check if we should use the same metrics names as in https://github.com/zalando/skipper/pull/2597

Reply:
done